cmd/yggctl/actions.go

```diff
@@ -165,14 +165,18 @@ func listenAction(ctx *cli.Context) error {
 			if !ok {
 				return cli.Exit(fmt.Errorf("cannot cat %T as uint32", s.Body[1]), 1)
 			}
+			messageID, ok := s.Body[2].(string)
+			if !ok {
+				return cli.Exit(fmt.Errorf("cannot cat %T as string", s.Body[2]), 1)
```

Comment:
There is a typo in this log message. Do you mind correcting it?
```suggestion
				return cli.Exit(fmt.Errorf("cannot cast %T as string", s.Body[2]), 1)
```

Reply:
Thanks for catching these! I've pushed the changes to fix these.